scripts/releng/update-release-info.sh

```diff
@@ -8,7 +8,7 @@ title: Kubernetes Release Cycle
 type: docs
 auto_generated: true
 ---
-<!-- THIS CONTENT IS AUTO-GENERATED via ./scripts/releng/update-release-info.sh in k/website -->
+<!-- THIS CONTENT IS AUTO-GENERATED via ./scripts/releng/update-release-info.sh in kubernetes/website -->
```

Comment:
I prefer either of:
- `k/website`
- https://github.com/kubernetes/website

`kubernetes/website` doesn't provide enough context for the uninitated, and is more typing for those in the know.